pkg/api/admin_provisioning.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/grafana/grafana/pkg/models"
 )
 
-func (server *HTTPServer) AdminProvisioningReloadDasboards(c *models.ReqContext) Response {
+func (server *HTTPServer) AdminProvisioningReloadDashboards(c *models.ReqContext) Response {
```

Comment:
I think this is private and only used by the plg/api/api.go mapping below which is unchanged.